geomstats/geometry/symmetric_matrices.py

```diff
@@ -0,0 +1,22 @@
+"""The vector space of symmetric matrices."""
+
+from geomstats.geometry.embedded_manifold import EmbeddedManifold
+from geomstats.geometry.matrices import Matrices
+
+EPSILON = 1e-6
+TOLERANCE = 1e-12
+
+
+class SymmetricMatrices(EmbeddedManifold):
+    """Class for the vector space of symmetric matrices of size n."""
+
+    def __init__(self, n):
+        assert isinstance(n, int) and n > 0
+        super(SymmetricMatrices, self).__init__(
+            dimension=int(n * (n + 1) / 2),
+            embedding_manifold=Matrices(n, n))
+        self.n = n
+
+    def belongs(self, mat, atol=TOLERANCE):
+        """Check if mat belongs to the manifold of symmetric matrices."""
```

Comment:
Although both are true, maybe consistently use vector space or manifold to describe the space

Reply:
Shouldn't it inherit from Matrices?

Reply:
That is all the question... If a class inherits from Matrices, it has to use its constructor which asks for two parameters m and n (size of the matrix). Then your class is considered as a manifold of dimension m*n since Matrices constructor defines dimension=m*n.

Reply:
For now, all matrix subgroups don't inherit from GeneralLinear so I consistently did not make the subvector space SymmetricMatrices inherit from Matrices. But I agree this should be modified.